timescale-forge/forge-multi-node.md

```diff
@@ -1,58 +1,58 @@
-# Setting up TimescaleDB 2.0 Multi-node on Timescale Forge
+# Setting up TimescaleDB 2.0 multi-node on Timescale Forge
```

Comment:
I wonder at this point if we should use something than `2.0` since we're about to release `2.3` on Forge, etc. Thoughts?

Reply:
Either `TimescaleDB 2` or `TimescaleDB 2.x` ?

Reply:
I'm a fan of either 2.x (when not version specific) or... how do we feel about `2.0+` or `>2.0` (maybe those are international friendly...)

Reply:
I would use `Timescale 2.0 or greater` if I was talking about pre-reqs or something, but in a heading I think I'd prefer `2.x` (the symbols are just confusing, and prone to breakage). If we want to take an SEO focus, we should probably have different pages for 2.0, 2.1, 2.2, ... but that might be a step too far 😉